cookie-notice.md

```diff
@@ -0,0 +1,93 @@
+
+Cookies that expire at the end of a browser session (normally when you exit your browser) are called "session cookies". Session cookies can be used to recognize and link your actions during a single browsing session. Cookies that can be stored for longer are called "persistent cookies", and can be used to remember your preferences or actions on the website or service that you are using.
+
+  
+
+### 2\. HOW WE USE COOKIES AND SIMILAR TRACKING TECHNOLOGIES
+
+We use first-party and third-party cookies for several reasons. Some cookies are required for technical reasons in order for our Services to work, and we refer to these as "strictly necessary" cookies. Some cookies measure traffic and statistical data about usage of our Services, which we use to understand how our Services are being used, and we refer to these as "performance" cookies. Other cookies enable us to personalize your experience or otherwise improve the functionality of the Services, and we refer to these as "functionality" cookies. Finally, we also use cookies to personalize content for you and make advertising and marketing messages more relevant to you, and we refer to these as "targeting" cookies.
+
+**Other tracking technologies:** We and our third-party partners may use other, similar technologies from time to time, like web beacons, software development kit (SDKs), pixels (or "clear gifs") and other tracking technologies. Pixels are tiny graphics files that contain a unique identifier that enable us to recognize when someone has visited our Services or, in the case of web beacons, opened an email that we have sent them. This allows us, for example, to monitor the traffic patterns of users from one page within our Services to another, to deliver or communicate with cookies, to improve site performance, and to measure the success of marketing campaigns. We also use browser web storage (including via HTML5), also known as local storage objects, for similar purposes as cookies. Local storage objects enable the storage of a larger amount of data than cookies. While you may not have the ability to specifically reject or disable these tracking technologies, in many instances, these technologies are reliant on cookies to function properly; accordingly, in those instances, declining cookies will impair the functionality of these technologies. Your web browser may also provide functionality to clear your browser web storage, see Section 5 below for more information.
+
+**We never use targeting cookies on our Platform aimed at Students. For Students using our Platform, we only use strictly necessary cookies and limited performance and functionality cookies in order to provide our Services to Users.**
+
+  
+
+### 3\. COOKIES WE SERVE THROUGH OUR SERVICES
+
+The first-party and third-party cookies served through our Services and the purposes they perform are described in more detail below.
+
+  
```

Comment:
hmm... so what should we have here. Should at the very least point to the dynamic html on the webpage?

Reply:
Can do - sounds good, since this is not a place to review the cookies, just the  change history of the legalese around the listing